.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,281 @@
+name: e2e-tests
+
+on:
+  push:
+    tags:
+      - '*tetrate*'
+
+jobs:
+  create-test-images:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+    name: create-test-images
+    runs-on: ${{ matrix.os }}
+    env:
+      HUB: hellozee-docker-istio-testing.bintray.io # change
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: docker-login
+        uses: docker/login-action@v1
+        with:
+          registry: hellozee-docker-istio-testing.bintray.io # change
+          username: ${{ secrets.BINTRAY_USER }}
+          password: ${{ secrets.BINTRAY_API_KEY }}
+
+      - name: build and push images
+        shell: 'script -q -e -c "bash {0}"'
+        run: |
+          export TAG=$(git describe --tags)
```

Comment:
This needs a cross check why $GITHUB_REF isn't working

Reply:
I think we need to get into the runner through shell and check out why this and creating a new branch providing another one doesn't work

Reply:
Is there update on this?

Reply:
Please create an issue - we need to address this.

Reply:
https://github.com/tetratelabs/getistio/issues/77